sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStore.scala

```diff
@@ -330,7 +330,8 @@ trait StateStoreProvider {
    *                         A value not greater than 0 means the operator doesn't activate prefix
    *                         key, and the operator should not call prefixScan method in StateStore.
    * @param useColumnFamilies Whether the underlying state store uses a single or multiple column
-   *                          families
+   *                          families; by default we'll use virtual column family if this parameter
```

Comment:
I feel like we shouldn't mention this - it's an implementation detail and even specific to RocksDB implementation. We define "interface" here and it's up to provider implementation how to deal with column families.